BowlingBall.Tests/GameTest.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+namespace BowlingBall.Tests
+{
+    public class GameTest
+    {
+        [Fact]
+        public void RollWithPinsMoreThan10()
+        {
+            //roll cannot be more than 10 or negative
```

Comment:
Create one private variable to hold instance of Game class and use that variable everywhere, rather than creating new game instance in every fixture